types/react-flatpickr/index.d.ts

```diff
@@ -4,13 +4,19 @@
 // Definitions: https://github.com/DefinitelyTyped/DefinitelyTyped
 // TypeScript Version: 2.3
 
-import { Component } from 'react';
-import { EventCallback, Options } from 'flatpickr';
+import { Component, DetailedHTMLProps, InputHTMLAttributes, HTMLAttributes } from 'react';
+import { Hook, Options } from 'flatpickr';
```

Comment:
[flatpickr](https://github.com/chmln/flatpickr/blob/8c55b50373e42e4eeb386112b5e0ec578298e229/index.d.ts) appears to now export its own TypeScript types and hence the ones in the DefinitelyTyped repo are now redundant.

So I think we need to do that first before we can proceed here (I'll open an issue for it).